### DISCUSSION TITLE
typeORM relations

### DISCUSSION THREAD

**Discussion by qckid98** [brocoders/nestjs-boilerplate#1663] [Q&A]
Hi, i have questions regarding relations on this boilerplate.

Is there any CLI to include relations to the resources? or should i add it manually? if manual, then i need to add it to resources.entity.ts only or any other files as well?

**Comment by Shchepotin** [comment]
@qckid98 currently you can add it manually.

You need to add it to *.entity.ts, domain, mappers, and DTO (if required)
